### PR TITLE
fix: add atc button to all product component in home, list, and recomendation

### PR DESCRIPTION
**Pull request by DiebrataA**
Home
image
image

List
image
image
image
image
image
image

List[slug]
image
image

### CODE REVIEW

**Comment by DiebrataA** [comment]
added POW for cart page
https://drive.google.com/drive/folders/1uZPYrEbdBtWO0cgydSTIKcFs_lXWegFk?usp=share_link

File: public/scss/components/ChooseVariant.module.scss [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    padding: 0;
+    background-color: $color_white;
+    border-radius: 0 0 3px 3px;
+
+    &__inner
+    {
+        @include fixedWidth(375px);
+        min-height: 48px;
+
+        @media screen and (max-width: 575px)
```
Comment (mayun19) [inline public/scss/components/ChooseVariant.module.scss:16]: pakai mixin varible ga

**Review by mayun19 — CHANGES_REQUESTED** [review]

File: pages/[lng]/index.tsx [hunk 2]
```diff
@@ -37,11 +44,51 @@ const Home: FC<any> = ({
...
+  useState<boolean>(false);
+  const [isAddToCart, setIsAddToCart] = useState<boolean>(false);
+  const [hasQuickViewFeature, setHasQuickViewFeature] = useState<boolean>(false);
+  const [showPopupSuccessAddCart, setShowPopupSuccessAddCart] = useState<boolean>(false)
+  const [showPopupSuccessNotify, setShowPopupSuccessNotify] = useState<boolean>(false)
+  const [showPopupErrorAddCart, setShowPopupErrorAddCart] = useState<boolean>(false)
+  const [showPopupErrorNotify, setShowPopupErrorNotify] = useState<boolean>(false)
+
+  // function
+  const tooglePopupSuccessAddCart = () => setShowPopupSuccessAddCart(!showPopupSuccessAddCart)
```
Comment (ardiankristo) [inline pages/[lng]/index.tsx:58]: Saran buat function yang berfungsi sbg state toggler (bolean state) menerapkan best practice nya yang seperti ini biar ngga ada kemungkinan race condition

https://stackoverflow.com/a/48394541/18537254

Jadi gini
`const tooglePopupSuccessAddCart = () => setShowPopupSuccessAddCart((prevState) => !prevState)`

note: nama variabel `prevState` bebas kalo mau diganti pake nama state nya juga bole

**Review by ardiankristo — CHANGES_REQUESTED** [review]

**Comment by DiebrataA** [comment]
Added Pop up success when add to cart in cart page
image

**Review by iqbalwijonarko — APPROVED** [review]

**Pull request merged**